hal/network/ncp_client/sara/sara_ncp_client.cpp

```diff
@@ -1033,6 +1020,10 @@ int SaraNcpClient::selectNetworkProf(ModemState& state) {
             }
             // KORE AT&T or 3rd Party SIM
             else {
+                // Hard code ATT for 05.12 firmware versions
+                if (fwVersion_ == UBLOX_NCP_R4_APP_FW_VERSION_0512) {
```

Comment:
There should still be a check for `if (netConf_.netProv() == CellularNetworkProvider::KORE_ATT)` specifically, as this block says 'KORE AT&T or **3rd Party SIM**'. Let's also make the same change for Gen 3 code to make them a bit more similar.

Reply:
Overlooked this - thanks. Added. 👍